collector/collector.go

```diff
@@ -206,6 +213,17 @@ loop:
 			if err != nil {
 				log.Error("cannot write message %v", err)
 			}
+		case data := <-tortoiseBeacons:
+			var e events.TortoiseBeaconCalculated
+			err := types.BytesToInterface(data[size:], &e)
+			if err != nil {
+				log.Error("cannot parse received message %v", err)
```

Comment:
wrong log

Reply:
return here? don't store beacon if we failed to deserialise

Reply:
@antonlerner Should I do the same fix for other `case`s in switch while I'm fixing it here?